lib/ansible/modules/net_tools/nios/nios_host_record.py

```diff
@@ -39,6 +39,16 @@
     default: default
     aliases:
       - dns_view
+  configure_for_dns:
```

Comment:
What is the reason for alias. How about renaming this option itself to `dns`

Reply:
I have kept the same name that we send in the json request body to infoblox client, as other variables are named and kept as same.